integrations/gitea-repositories-meta/user27/template1.git/hooks/pre-applypatch.sample

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#
+# An example hook script to verify what is about to be committed
+# by applypatch from an e-mail message.
+#
+# The hook should exit with non-zero status after issuing an
+# appropriate message if it wants to stop the commit.
+#
+# To enable this hook, rename this file to "pre-applypatch".
+
+. git-sh-setup
+precommit="$(git rev-parse --git-path hooks/pre-commit)"
```

Comment:
Please notice that this is not `sh` syntax, `sh` requires back ticks. In Linux, `bash` is usually (always?) aliased as `sh` for convenience, as it is considered a superset of it. If we don't target unices without this default, then this is OK.

Reply:
This is the same as the other meta repos, and is only used for tests.

Reply:
Yes, of course, it's just a `.sample`. 😅